libvast/src/system/configuration.cpp

```diff
@@ -148,8 +188,9 @@ caf::error configuration::parse(int argc, char** argv) {
   std::vector<std::string> caf_args;
   std::move(caf_opt, command_line.end(), std::back_inserter(caf_args));
   command_line.erase(caf_opt, command_line.end());
-  // Instead of the CAF-supplied `config_file_path`, we use our own
-  // `config_files` variable in order to support multiple configuration files.
+  // Collect config files. Instead of the CAF-supplied `config_file_path`, we
```

Comment:
We still need to add a `VAST_CONFIG` variable to map onto the `vast --config=` option. This is handled manually below.

Reply:
Yep, I added a one-off...alas.